code/game/objects/effects/contraband.dm

```diff
@@ -350,7 +350,7 @@
 
 /obj/structure/sign/poster/contraband/free_drone
 	name = "Free Drone"
-	desc = "This poster commemorates the bravery of the rogue drone banned by CentComm."
+	desc = "This poster commemorates the bravery of the rogue drone remotely detonated by Centcom."
```

Comment:
The reason I suggest `"destroyed" is because this poster is illegal rebellious contraband praising a rogue maintenance drone, and it sounds more heroic if you say destroyed.